packages/injective-math/src/fp_decimal/exp.rs

```diff
@@ -51,6 +51,29 @@ impl FPDecimal {
         }
         val
     }
+
+    // a^(0.5)
+    pub fn _root(a: FPDecimal) -> Option<FPDecimal> {
```

Comment:
if it's a function meant for external use, you should call it `root`. And maybe `sqrt` is better?